src/footer/index.js

```diff
@@ -0,0 +1,34 @@
+import styles from './styles.css';
+
+const listNumbers = (from, to) => {
+  const list = [];
+
+  for (let i = from; i <= to; i++) {
+    list.push(i);
+  }
+
+  return list;
+};
+
+const ListOfNumbers = ({
+  from, to, odd, even
+}) => (
+  <ul className={styles.ul}>
+    {
+      even ? listNumbers(from, to)
+        .filter((number) => number % 2 === 0)
+        .map((number) => <li className={styles.li} key={number}>{number}</li>)
+        : odd ? listNumbers(from, to)
+          .filter((number) => number % 2 !== 0)
+          .map((number) => <li className={styles.li} key={number}>{number}</li>)
+          : listNumbers(from, to)
+            .map((number) => <li className={styles.li} key={number}>{number}</li>)
+    }
```

Comment:
it's very difficult to read. Do it via `if`. 
`map` part is the same in every case, so what the reason to repeat it 3 times